test/integration/webhook.test.ts

```diff
@@ -1,6 +1,6 @@
 import fs from 'fs'
 import request from 'supertest'
-import { createProbot, Probot } from '../..'
+import { createProbot, Probot } from '../../src'
```

Comment:
Not 100% sure why TypeScript complained about this module resolution string in this PR but not #695 🤔